libs/ui-lib/lib/ocm/index.ts

```diff
@@ -1,4 +1,3 @@
-export * as Api from './api';
```

Comment:
This breaks the uhc-portal integration...

Reply:
Perhaps change it like this:
```
export * as Api from '../common/api';
```

Reply:
Pay attention to line #15

Reply:
I've changed it

Reply:
Remove line [16](https://github.com/openshift-assisted/assisted-installer-ui/pull/2429/files#diff-bbfc2ed660cb37b5cb6cc5a67fc7b6b5e8424611efb7af8d7c3d303c76abd895R16). It is injecting all the stuff grouped into `Api` into this module, so you end up with twice the same objects!!!

Reply:
ok, I forgot it